package.json

```diff
@@ -62,20 +66,28 @@
     "ember-load-initializers": "^2.1.2",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-noscript": "^4.1.0",
-    "ember-qunit": "^4.6.0",
+    "ember-page-title": "^6.2.1",
```

Comment:
This shouldn't be added (or it should be disabled somehow) we don't want to control the page title in an LTI container. There is also a helper here that noops this.